storm-multilang/dotnet/src/main/dotnet/Dotnet.Storm.Adapter/Test/TestAPI.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using CommandLine;
+using Dotnet.Storm.Adapter.Channels;
+using Dotnet.Storm.Adapter.Components;
+using Dotnet.Storm.Adapter.Logging;
+using Dotnet.Storm.Adapter.Messaging;
+using Dotnet.Storm.Adapter.Serializers;
+using log4net;
+using log4net.Config;
+using log4net.Core;
+
+namespace Dotnet.Storm.Adapter.Test
+{
+    public static class TestAPI
+    {
+        public static Component CreateComponent(Type type)
+        {
+            Channel.Instance = new CacheChannel()
+            {
+                Serializer = new JsonSerializer()
```

Comment:
It looks the Serializer is not used.. Remove it unless there is a need to use it

Reply:
Updated